v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -89,6 +89,8 @@ spec:
          - exposedPort: 13132
          - exposedPort: 13133
      memoryLimit: "512M"
+     cpuLimit: "1000m"
+     cpuRequest: "500m"
```

Comment:
This is probably too high. Normally theia container uses 30m. If we set 500 as cpu request we are going to prevent other processes to use the rest of the memory, even if Theia is inactive, because we have blocked 500m for theia bootstrap.

Reply:
well, we can stick to `0.03` and `1`, but my understanding that the request is smth. that is guaranteed by k8s and since currently che-theia IDE is the most important container in the workspace pod, we might like to allocate more to guarantee the
proper functionality and prevent potential throttling

Reply:
> Normally theia container uses 30m

Where do you get this number from? IMO, the theia back-end uses more than 3% of a single core as soon as you're doing something.